Automattic/woocommerce-payments
# feat: save express payment options

Helps with https://github.com/Automattic/woocommerce-payments/issues/1905

#### Changes proposed in this Pull Request









Splitting up a bigger PR.
This ensures that the settings are saved to the backend and correctly displayed in the settings screen.

I will ensure that they are correctly displayed on the frontend (during checkout) on a separate PR.
There's already a separate task that ensures that the settings are migrated: https://github.com/Automattic/woocommerce-payments/issues/1991

#### Testing instructions





- Go to http://localhost:8082/wp-admin/admin.php?page=wc-settings&tab=checkout&section=woocommerce_payments_digital_wallets
- Page loads with some default
- Change some values
- Save
- Refresh the page
- The page should be refreshed with the newly saved values

-------------------
